features/FEATURE_BLE/source/generic/GenericSecurityManager.cpp

```diff
@@ -500,6 +500,25 @@ ble_error_t GenericSecurityManager::requestAuthentication(connection_handle_t co
 // MITM
 //
 
+ble_error_t GenericSecurityManager::generateOOB(
+    const address_t *address
+) {
+    /* legacy pairing */
```

Comment:
It would be good to return an error if OOB is already being generated. No reason to override the address of the current request.

Reply:
addressed below